protoc-gen-bq-json-schema/genjson/config.go

```diff
@@ -0,0 +1,11 @@
+package genjson
+
+import "github.com/spf13/pflag"
+
+type Config struct {
+	Indent bool
+}
+
+func (c *Config) AddToFlagSet(flags *pflag.FlagSet) {
```

Comment:
really like this way of managing flags - nice!

Reply:
Yeah, it's a copy-paste from https://github.com/einride/protoc-gen-go-cli 😉